enhancements/hypershift/node-tuning.md

```diff
@@ -0,0 +1,201 @@
+1. The containerized TuneD daemon (operand) will calculate the kernel boot parameters that should be set on the node according to the admin defined TuneD profile (if any), and will write the kernel boot parameters to the Profile status for the Profile object corresponding to the node on which the daemon is running (same as standalone OCP). 
+2. NTO will generate a MachineConfig for setting the calculated kernel boot parameters to the whole NodePool and will embed the MachineConfig into a ConfigMap. This ConfigMap will have a specific label to specify that it is an NTO-generated MachineConfig, and another label for the NodePool name that it corresponds to.
+    - Similar to a limitation in standalone OCP with MachineConfigPools, at most one MachineConfig will be generated per NodePool by NTO. It will be documented that if users wish to apply different kernel boot parameters to Nodes, they cannot be in the same NodePool.
+3. The NodePool controller will need to watch for these NTO generated ConfigMaps in the hosted control plane namespace and reconcile the corresponding NodePool if one is created / updated.
+    - There will be at most one NTO-generated MachineConfig per NodePool, and they will only change based on admin changes to the MachineConfig or (sometimes) after a cluster upgrade has completed and the containerized TuneD daemon has recalculated the kernel boot parameters on the upgraded RHCOS.
+
+
+### API Extensions
+We will add a field to the NodePool `spec` called `tuningConfig`. This field will contain a list of object references to ConfigMaps. This field will be used to specify a list of ConfigMaps containing Tuned object manifests.
+
+### Risks and Mitigations
+- Since Tuned CRs will exist in the hosted cluster, an admin user of this cluster could modify the Tuned objects directly, giving them the ability to change the OS-level tunings.
+  - We mitigate this issue by making the Node Tuning Operator sync these Tuneds with the ConfigMaps in the hosted control plane namespace before calculating the profile for the containerized TuneD daemon to apply.
+
+### Drawbacks
+N/A.
+
+### Test Plan
+N/A.
+
+#### Dev Preview -> Tech Preview
+#### Tech Preview -> GA
+#### Removing a deprecated feature
+N/A as we are going straight to GA.
+
+### Upgrade / Downgrade Strategy
+In general, the upgrade will be very similar to standalone cluster, just the operator will not be started by CVO, but by control-plane-operator. 
+In phase 3, NTO-generated MachineConfigs may change after an upgrade, but not during an upgrade. The MachineConfigs would change if the kernel or TuneD daemon version changed on the Node in a way that resulted in different calculated kernel boot parameters. This would not happen until NTO is upgraded and rolls out the containerized TuneD DaemonSet.
+
+### Version Skew Strategy
+N/A.
+
+### Operational Aspects of API Extensions
+N/A.
+
+#### Failure Modes
+
+#### Support Procedures
+
+## Alternatives
```

Comment:
can we have a section in this doc to future exploring moving all CRDs into management side and run pod inly ephemerally in the hos4ed cluster?